api/XPathExpression.json

```diff
@@ -23,10 +23,10 @@
             "version_added": false
           },
           "opera": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
I'm pretty sure this was supported in Presto. `XPathEvaluator` is the strangest thing, both an interface and mixed into `Document` at the same time. So there's actually a `document.createExpression()` method too, which our data says was supported in Presto.

Can you update the custom tests for this to try creating an instance with both methods?

Reply:
Ah, looks like there's a slight issue with the test, so it generated a range and I probably accidentally ignored it.  I double-checked in BrowserStack and confirmed Presto support!

Reply:
Thanks!